webapp/src/components/rhs/rhs_checklist_list.tsx

```diff
@@ -394,13 +405,40 @@ const StyledPrimaryButton = styled(PrimaryButton)`
     margin: 12px 0;
 `;
 
+const OverdueTasksToggle = styled.div<{toggled: boolean}>`
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    display: inline-block;
+    margin-left: 5px;
+    padding: 2px 4px;
+    align-items: center;
+    border-radius: 4px;
+    user-select: none;
+    background-color: ${(props) => (props.toggled ? 'var(--dnd-indicator)' : 'rgba(var(--dnd-indicator-rgb), 0.08)')};
+    color: ${(props) => (props.toggled ? 'var(--button-color)' : 'var(--dnd-indicator)')};
+`;
+
 export default RHSChecklistList;
 
+const overdueTasks = (checklists: Checklist[]) => {
+    let count = 0;
+    const now = DateTime.now();
+    for (const list of checklists) {
+        for (const item of list.items) {
+            if (item.due_date > 0 && DateTime.fromMillis(item.due_date) <= now) {
```

Comment:
The second suggestion, that I somehow forgot to add before:
Should we count only unchecked items here?
```suggestion
            if (item.state !== ChecklistItemState.Open && item.state !== ChecklistItemState.InProgress && item.due_date > 0 && DateTime.fromMillis(item.due_date) <= now) {
```

Reply:
Yes, we are not interested in finished tasks. We want to show number of tasks that are open/in progress and are overdue.

Reply:
Ugh, sorry, my suggestion above was wrong, I should have used `===` instead of `!==`. I tested the PR and overdue *checked* tasks are still counted, so I think we still need to fix this, right?
This is what I meant before, sorry for the confusion:
```suggestion
            if ((item.state === ChecklistItemState.Open || item.state === ChecklistItemState.InProgress) && item.due_date > 0 && DateTime.fromMillis(item.due_date) <= now) {
```

Reply:
Oh, now I see what you meant. Nice catch! Fixed it.